SmartDeviceLink/SDLFileManager.m

```diff
@@ -204,6 +209,10 @@ - (void)sdl_listRemoteFilesWithCompletionHandler:(SDLFileManagerListFilesComplet
     __weak typeof(self) weakSelf = self;
     SDLListFilesOperation *listOperation = [[SDLListFilesOperation alloc] initWithConnectionManager:self.connectionManager completionHandler:^(BOOL success, NSUInteger bytesAvailable, NSArray<NSString *> *_Nonnull fileNames, NSError *_Nullable error) {
```

Comment:
Instead of checking the description, the operation should return the `resultCode` in the error `userInfo`.

Reply:
fixed